### DISCUSSION TITLE
Robot path on map

### DISCUSSION THREAD

**Discussion by shagadeli** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#716] [Q&A] [answered]
I have a Dreame L10s Ultra Pro Heat set up with the Tasshack/dreame-vacuum integration, the map and commands work great, but there is no robot position or path in the map. Is this unavailable or should it be activated somewhere?

**Comment by PiotrMachowski** [comment, marked as answer]
This question should be asked in https://github.com/Tasshack/dreame-vacuum repo

**Reply by shagadeli** [reply]
Will do. Thanks
